symFileManager.py

```diff
@@ -57,30 +158,30 @@ def GetLibSymbolMap(self, libName, breakpadId):
     if libSymbolMap is None:
       LogDebug("Need to fetch PDB file for " + libName + " " + breakpadId)
 
-      # Guess the name of the .sym file on disk
-      if libName[-4:] == ".pdb":
-        symFileName = re.sub(r"\.[^\.]+$", ".sym", libName)
-      else:
-        symFileName = libName + ".sym"
-
-      pathSuffix = os.path.join(libName, breakpadId, symFileName)
-      urlSuffix = "/".join([libName, breakpadId, symFileName])
+      libSymbolMap = self.diskCache.fetch(pathSuffix)
```

Comment:
rest of the file is using capitals for the first letter of the function name, i.e. Fetch